.github/workflows/ci.yml

```diff
@@ -14,6 +14,7 @@ concurrency:
 env:
   NX_CLOUD_ACCESS_TOKEN: ${{ secrets.NX_CLOUD_ACCESS_TOKEN }}
   YARN_ENABLE_HARDENED_MODE: 0
+  REPLAY_API_KEY: ${{ secrets.REPLAY_API_KEY || 'rwk_mRDHS8s5EA5SrID1k2Xqu5hKJumF8LzUTSItmLw9Cto' }}
```

Comment:
To be completely honest with you, this doesn't feel great 😕
Is there really no way to *not* have an api key visible for everyone to see?

Reply:
I'm talking internally with the team to improve this process. Some sort of "warn missing key and continue without replay" solution. I'll follow up when we do that.

Reply:
To be clear: it's because of the security design of GitHub action that doesn't share secrets with forks.